webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/Components/WebhookDeleteModal.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import ForemanModal from 'foremanReact/components/ForemanModal';
+import { foremanUrl } from 'foremanReact/common/helpers';
+
+import { WEBHOOK_DELETE_MODAL_ID } from '../../constants';
+
+const WebhookDeleteModal = props => {
+  const { id, name } = props.toDelete;
```

Comment:
Can you replace it with 
`const WebhookDeleteModal = ({toDelete, onSuccess}) =>`